src/main/features/claimant-response/draft/draftClaimantResponse.ts

```diff
@@ -22,12 +23,13 @@ export class DraftClaimantResponse extends DraftDocument {
   alternatePaymentMethod?: DraftPaymentIntention
   courtCalculatedPaymentIntention?: PaymentIntention
   courtOfferedPaymentIntention?: PaymentIntention
-  courtDecisionType?: DecisionType
+  decisionType?: DecisionType
   freeMediation?: FreeMediation
   paidAmount?: PaidAmount
   rejectionReason?: RejectionReason
   acceptCourtOffer?: AcceptCourtOffer
   disposableIncome: number
+  courtDetermination?: CourtDetermination
```

Comment:
Careful here, you are mixing domains. `CourtDetermination` is not a form model but part of the claim model. Therefore can't be reused here. You need to create your own form model.

Reply:
Also in your new form model make sure `decisionType` is an enum (not a string like it is now in the claim model). That enum must also be a new one defined as a form model (so you can't reuse `DecisionType` from `common/court-calculations/courtDecision.ts` or you will contaminate domain models again).

Reply:
used correct domain models now..